setup1-shared/src/structures.rs

```diff
@@ -12,6 +12,17 @@ pub enum SetupKind {
     Universal,
 }
 
+impl SetupKind {
+    pub fn as_string(&self) -> String {
+        match self {
+            SetupKind::Development => "Development".to_owned(),
```

Comment:
if we use lowercase strings we may stay compatible with strings produced by serde